packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -99,7 +104,7 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
         target={link.target}
         rel={rel}
         disabled={link.disabled}
-        aria-label={link.ariaLabel}
+        aria-label={`${ariaLabel} ${selectStateAria}`}
```

Comment:
You will be having at least a space character aria-label if both are not provided, an `ariaLabel` with a space at the end if the `selectAriaState` missing or a `selectAriaState` with a space at the beginning if `ariaLabel` missing. This is actually reflected in your snapshot updates. Perhaps a ternary would be helpful in here:
```suggestion
        aria-label={ariaLabel + selectStateAria ? ` ${selectStateAria}` : ‘’}
```